HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -97,6 +97,7 @@ def self.apply_air_source_hvac_systems(model, runner, cooling_system, heating_sy
         htg_coil.setNominalCapacity(UnitConversions.convert(heating_system.heating_capacity, 'Btu/hr', 'W'))
         htg_coil.setName(obj_name + ' htg coil')
         htg_coil.additionalProperties.setFeature('HPXML_ID', heating_system.id) # Used by reporting measure
+        htg_coil.additionalProperties.setFeature('IsSupplementalCoil', heating_system.is_heat_pump_backup_system) # Used by reporting measure
```

Comment:
Trying to make the same change to all appropriate objects, and ensure new objects in the future are covered, feels a bit brittle. It seems like a more robust solution would be, in the `get_object_output_variables_by_key` method of the reporting measure, to retrieve the HPXML_ID that is already being set on the OSM object, and then check if the HPXML object with that ID is/has a backup system. (For a HPXML HeatingSystem object that serves as backup, you can use [this method](https://github.com/NREL/OpenStudio-HPXML/blob/d3a3bd73ba2c6a398fd722826b608ce5bbe20ba9/HPXMLtoOpenStudio/resources/hpxml.rb#L3239) to check.)